tracer/src/Datadog.Trace.Tools.Runner/CheckIisCommand.cs

```diff
@@ -0,0 +1,147 @@
+// <copyright file="CheckIisCommand.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
```

Comment:
Not sure how much of this you have done elsewhere, just a suggestion:

```suggestion
#nullable enable
```

Reply:
I tried, I can't use it in the commands. There is... a conflict.. of some kind. I don't want to spoil the fun, I will let you try when you're bored 😁